node.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+    'extends': [
+        'airbnb-base',
+        './.eslintrc-base.js',
+    ],
+    'globals': {
```

Comment:
should this be environments? https://eslint.org/docs/user-guide/configuring#specifying-environments